pkg/plugin/javascript/dashboard_delete.go

```diff
@@ -44,6 +44,16 @@ func (d *DashboardDelete) Call(ctx context.Context, vm *goja.Runtime) func(c goj
 		// This will never error since &key is a pointer to a type.
 		_ = vm.ExportTo(obj, &key)
 
+		if len(c.Arguments) > 1 {
```

Comment:
This is needed to keep the metadata argument optional. I don't have strong opinions about this so let me know if anybody prefers this to always be passed with calls to the data store.

Reply:
Javascript [doesn't care one way or the other](https://dmitripavlutin.com/javascript-function-parameters/), so either way seems fine to me.

Reply:
I was thinking of a scenario where octant users decide to use the older plugins with new octant release. This will not introduce any breaking change in that case